packages/visualizations/src/components/Table/Cell/Format/DateFormat.svelte

```diff
@@ -1,22 +1,26 @@
 <script lang="ts">
+    import { getTableContext } from '../../context';
     import type { DateColumn } from '../../types';
+
     import { warn } from './utils';
 
     export let rawValue: unknown;
     export let options: DateColumn['options'];
 
-    function getDisplayValue(v: unknown) {
+    const context = getTableContext();
+
+    function getDisplayValue(v: unknown, locale: string) {
         if (
             (typeof v === 'string' || typeof v === 'number') &&
             !Number.isNaN(new Date(v).getTime())
         ) {
-            return new Intl.DateTimeFormat(navigator.language, options).format(new Date(v));
+            return new Intl.DateTimeFormat(locale, options).format(new Date(v));
```

Comment:
Could default to navigator.language? Not useful for us—almost a bit dangerous if we forget to set it somehow—but could be nice.

Reply:
Currently with this PR the default locale is `navigator.language` (see context.ts). 
Indeed for our use case, we'll always give a `locale` prop. In we don't setup navigator.language as default locale, which value should we set ? a hardcoded locale 'en' ?

Reply:
Yes, I missed the context part early in this review, indeed, sorry! It's alright if it defaults to `navigator.language`👌